### PR TITLE
Update email and author

### PR DESCRIPTION
**Pull request by pvk-developer**

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -45,9 +45,8 @@ requirements:
 
 about:
   home: "https://sdv.dev"
-  license: MIT
```
Comment (amontanez24) [inline conda/meta.yaml:48]: I think we can just delete this file. We don't do these conda releases anymore

Comment (npatki) [reply conda/meta.yaml:48]: +1 we could just delete the entire `conda/` folder while we're at it right? Seems we did this for [Copulas](https://github.com/sdv-dev/Copulas) already.

**Review by npatki — CHANGES_REQUESTED** [review]
Let's delete the `conda/` folder altogether.

**Review by npatki — APPROVED** [review]
LGTM!

**Review by amontanez24 — APPROVED** [review]
:shipit:

**Pull request merged**